modules/mpv/testbeds/uosc.nix

```
@@ -0,0 +1,19 @@
+{ lib, pkgs, ... }:
+let
+  package = pkgs.mpv;
+in
+{
+  stylix.testbed.application = {
+    enable = true;
+    name = "mpv";
+    inherit package;
+  };
+
+  home-manager.sharedModules = lib.singleton {
+    programs.mpv = {
+      enable = true;
+      inherit package;
+      scripts = with pkgs.mpvScripts; [ uosc ];
```

This might be neater:

```suggestion
      scripts = [pkgs.mpvScripts.uosc];
```